### ISSUE TITLE
Review and Implement 'Accessibility Audit 2022' Bugs

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-reso-api#319]
Description

This ticket is responsible for analyzing the Accessibility Audit 2022 bugs that were identified back in October 2022 (see tickets linked to EPIC) and implementing the small/easy fixes. Any large/complex bugs should be scoped/estimated and discussed with Jessica prior to implementing.

Acceptance Criteria:

Review accessibility tickets attached BRS-995: Accessibility Audit 2022
TO DO
 + see if they are still valid

If it is valid and a quick code fix, then resolve - if not (fix is big) keep ticket open

Notes

**Comment by kgorana** [comment]
**Ashley Dhillon** 
April 11, 2023 at 11:43 AM

@Jessica Wade its not possible for EPIC’s to show in our backlog as a separate ticket due to the team-managed Jira project type we have setup for our board so I’ve created this ‘spike’ ticket to cover off the implementation part of the accessibility audit bugs in the backlog.
